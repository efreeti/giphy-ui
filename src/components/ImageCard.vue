<template>
  <div class="image-card">
    <span class="image-background">
      <slot />
    </span>
  </div>
</template>

<script lang="ts">
import { Component, Vue } from 'vue-property-decorator';

@Component
export default class ImageCard extends Vue {
}
</script>

<style lang="scss" scoped>
@import "~vue-material/dist/theme/engine";

.image-card {
  padding: 10px;

  text-align: center;
}

.image-background {
  display: inline-block;

  background-color: md-get-palette-color(grey, 200);
}
</style>
