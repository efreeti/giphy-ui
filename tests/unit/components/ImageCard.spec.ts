import { mount } from '@vue/test-utils';

import '../config';

import ImageCard from '@/components/ImageCard.vue';

describe('ImageCard', () => {
  it('should render correctly', () => {
    expect(mount(ImageCard, {
      slots: {
        default: '<div></div>',
      },
    }))
        .toMatchSnapshot();
  });
});
