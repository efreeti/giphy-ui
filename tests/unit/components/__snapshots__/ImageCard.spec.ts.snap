// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageCard should render correctly 1`] = `<div class="image-card"><span class="image-background"><div></div></span></div>`;
